infra.bs

```diff
@@ -290,8 +296,8 @@ the <a>string</a> with their corresponding <a>code point</a> in <a>ASCII upper a
 
 <hr>
 
-<p>To <dfn export>strip newlines</dfn> from a <a>string</a>, remove any U+000A LINE FEED and
-U+000D CARRIAGE RETURN <a>code points</a> from the <a>string</a>.
+<p>To <dfn export>strip newlines</dfn> from a <a>string</a>, remove any U+000A LF and U+000D CR
```

Comment:
Ah yeah I misread as U+009A.

I'm not sure what the = sign means. For non-control characters it seems to lead to a very verbose description. But I guess we could say "one of the names in that list"?